Resources/config/routing_pjax.yml

```diff
@@ -147,7 +147,7 @@ admin_roleList:
         _controller: ezsystems.platformui.controller.role:listRolesAction
 
 admin_roleView:
-    path: /role/view/{roleId}
+    path: /role/view/{roleId}/{tabId}
```

Comment:
instead of passing `null` in https://github.com/ezsystems/PlatformUIBundle/pull/438/files#diff-84de834a31786fc3fe688bc4535d6bdeR326 or 0 in https://github.com/ezsystems/PlatformUIBundle/pull/438/files#diff-77c6e07c3a54f32d5394f5241e4953b5R34 as `tabId` I think you should make this parameter optional